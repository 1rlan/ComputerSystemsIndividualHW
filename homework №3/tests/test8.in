6734573457361457316457961345973645635417
