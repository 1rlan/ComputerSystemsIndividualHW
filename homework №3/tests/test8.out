18884407562325.777388
